regression/tests/studio/test_studio_grading.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test studio grading
+"""
+from bok_choy.web_app_test import WebAppTest
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.grading_studio import GradingPageExtended
+
+from regression.tests.helpers import LoginHelper, get_course_info
+
+
+class StudioGradingTest(WebAppTest):
+    """
+    Test studio grading
+    """
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioGradingTest, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.grading_page = GradingPageExtended(
```

Comment:
Nit: It's a bit clearer if you declare all the page objects at the top of the class.